CMakeLists.txt

```diff
@@ -66,11 +70,11 @@ elseif(show_time_tool_msg)
     message(STATUS "Disable time_tool as getopt_long is not available")
 endif()
 
-if (BUILD_EXAMPLES)
+if (CCTZ_BUILD_EXAMPLES)
   add_subdirectory(examples)
 endif()
 
-if (BUILD_TESTING)
+if (CCTZ_BUILD_TESTING AND BUILD_TESTING)
```

Comment:
I think it would be better to use only CCTZ_BUILD_TESTING, it's better if there is only one way to do things.

Reply:
Any action here, or has this been abandoned?  I'm happy to merge if the CMake experts agree.